src/app/design/frontend/base/default/template/payment/form/omise/omiseoffsiteinternetbankingpayment.phtml

```diff
@@ -56,8 +57,11 @@ $code = $this->getMethodCode();
             </div>
             <div class="omise-banking-text-wrapper">
                 <span class="title">Bangkok Bank</span><br/>
-                <span class="rate secondary-text">Fee: 10 THB (same zone), 20 THB (out zone)</span>
+                <!-- <span class="rate secondary-text">Fee: 10 THB (same zone), 20 THB (out zone)</span> -->
             </div>
         </label>
     </li>
+    <li>
+        <span class="rate secondary-text"><?php echo($feeMessage); ?></span>
```

Comment:
why we just don't echo that:
$this->__('The bank may charge a small fee for the transaction.');
instead of setting it to variable used only once.

or do better: 
`<span class="rate secondary-text"><?= $this->__('The bank may charge a small fee for the transaction.'); ?></span>`
btw. is that ok.. for php5? `<?= ?>`

Reply:
I was trying to keep any PHP code as separate as possible, but thinking about it, it's probably better to have the text actually where it appears in the HTML - better for editing